docs/movie.sql

```diff
@@ -0,0 +1,61 @@
+create table member(
+                       id bigint auto_increment primary key,
+                       userId varchar(100) not null,
```

Comment:
```suggestion
                       userId varchar(100) unique not null,
```
중복 아이디 가입 안되게 막아야 할 것 같아용